.github/workflows/releases.yml

```diff
@@ -0,0 +1,61 @@
+name: Wheels
+
+on: [push, pull_request]
+
+jobs:
+  build_artifacts:
+    name: Build wheel on ubuntu-latest
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    env:
+      CIBW_TEST_COMMAND: pytest --pyargs zarr
+      CIBW_TEST_REQUIRES: pytest
+      CIBW_SKIP: "*27* pp*"
+
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
+
+      - uses: actions/setup-python@v1
+        name: Install Python
+        with:
+          python-version: '3.8'
+
+      - name: Install PyBuil
+        run: |
+          python -m pip install build
+
+      - name: Build wheel and sdist
+        run: |
+          python -m build
+      - uses: actions/upload-artifact@v1
+        with:
+          name: releases
+          path: dist
+
+  test_dist_pypi:
+    needs: [build_artifacts]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v1
+        with:
+          name: releases
+          path: dist
+
+      - name: test
+        run: |
+          ls
+          ls dist
+
+  upload_pypi:
+    needs: [build_artifacts]
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/v')
+    steps:
+      - uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
```

Comment:
Just out of curiosity, what is this? In other projects, we use `{{ secrets.pypi_username }}`.

Reply:
PyPI let you generate tokens for single projects. https://pypi.org/help/#apitoken